data/macos.yaml

```diff
@@ -518,6 +550,26 @@ sources:
 labels: [System]
 supported_os: [Darwin]
 ---
+name: MacOSNotificationCenter
+doc: macOS NotificationCenter database
```

Comment:
Instead of `macOS` I opt to use the more generic `MacOS`, unless it is specific to versions of MacOS that are named `macOS` ;)

Reply:
Correct, it was a typo.